client/src/components/Applications/AppForm/AppForm.jsx

```diff
@@ -562,6 +655,16 @@ export const AppSchemaForm = ({ app }) => {
               }
             )
           );
+          // Before job submission, ensure the memory limit is not above queue limit.
+          const queue = getExecSystemFromId(
+            app,
+            values.execSystemId
+          ).batchLogicalQueues.find(
+            (q) => q.name === values.execSystemLogicalQueue
+          );
+          if (app.definition.jobAttributes.memoryMB > queue.maxMemoryMB) {
```

Comment:
VM's don't have queues, so these queue values will be null

Reply:
> VM's don't have queues, so these queue values will be null

good catch, will handle that.

Reply:
Fixed and tested on https://cep.test/workbench/applications/opensees-express?appVersion=3.6.0